server/src/models/users.ts

```diff
@@ -3,3 +3,11 @@ import knex from "../pool";
 export async function getUserFromEmail(email: string) {
   return knex("users").where({ email }).first();
 }
+
+export async function checkUserID(id: number) {
+
+  return knex("users")
+  .select("id")
+  .where("id", id)
+  .then(rows => rows.length>0)
```

Comment:
```suggestion

  const rows =  await knex("users")
  .select("id")
  .where("id", id);
  
  return rows.length > 0;
```

Using `async` and promises together is a little... unnecessary? When you declare a function as `async` it will inherently return a promise. Here you declare a function as async, then immediately return a promise, rather than taking advantage of `await`.

Reply:
I think I resolve the promise in the calling function by using await which is why I chose to just return the promise

Reply:
I added the changes, thanks